### ISSUE TITLE
The Pinterest share page remains blocked when sharing a saved screenshot

### ISSUE THREAD

**Issue by Softvision-CristinaBadescu** [mozilla-services/screenshots#3885]
[Affected versions]:
- Firefox trybuild https://goo.gl/zWBXfk
- Screenshots 25.0.0

[Affected Platforms]:
- All Windows
- All Mac
- All Linux

[Prerequisites]:
- Have this trybuild: https://goo.gl/zWBXfk , installed on a new Firefox profile.
- Have one saved shot using "Firefox Screenshots" from the trybuild.
- Have a valid Pinterest account.

[Steps to reproduce]:
1. Open Firefox browser and log into the Pinterest account.
2. Click on the "View history, saved bookmarks, and more" button from the toolbar.
3. Click the "Screenshots" button and hover over the saved screenshot.
4. Click on the "Share" button and select the "Pinterest" icon.
5. From the Pinterest Share page choose a board and click the "Save" button.
6. Observe the behavior.

[Expected result]:
- The shot is successfully shared and after ~3 seconds the tab is closed and the saved shot preview page is focused.

[Actual result]:
- The "Save" button from the Pinterest share page is stuck in an infinite loop and the tab remains open.

[Notes]:
- The saved screenshot is shared on the selected Pinterest board, even though the Share page remains blocked.
- This issue is also reproducible on latest Firefox release 57.0.1 using Firefox Screenshots 19.2.0 and latest Nighlty build59.0a1 (2017-12-05) using Firefox Screenshots 25.0.0.
- Here is a screen recording with the issue:
![pinterest share page](https://user-images.githubusercontent.com/31244235/33610584-8d2b8980-d9d4-11e7-9764-74d432e5b47b.gif)

**Comment by ianb** [comment]
@Softvision-CristinaBadescu : I'm not able to reproduce this currently. I went to screenshots.firefox.com/shots and shared directly from there, and was able to save a shot successfully. (Could something have been wrong with Pinterest at the time?)

**Comment by Softvision-CristinaBadescu** [comment]
Hey @ianb, I also cannot reproduce this issue anymore. I've retested it using Firefox Screenshots 28.0.0 on latest Nightly build 59.0a1 (2018-01-05), on Windows 10 x64, Mac 10.13 and Arch Linux 4.12. I've managed to successfully share the saved screenshot, and the Pinterest share page didn't remained blocked. So yes, it seems like it was a platform issue.

Considering this, I'm going to close this one. Thanks.
